acquisitions/src/main/resources/thunderjet/mod-orders/features/independent-acquisitions-unit-for-ordering-and-receiving.feature

```diff
@@ -0,0 +1,645 @@
+    Prepare user that has only acqUnit1, try to edit related piece - operation should be forbidden
+    because for any operation with piece we check acq unit from related Title, and user does not have acqUnit2 assigned.
+
+    # 1. Create a composite order. Now 'acqUnitId1' is assigned to user, acqUnitId2' is not assigned to user
+    * configure headers = headersUser
+    Given path 'orders/composite-orders'
+    And request
+    """
+    {
+      id: '#(orderId3)',
+      vendor: '#(globalVendorId)',
+      orderType: 'One-Time',
+      acqUnitIds: ['#(acqUnitId1)']
+    }
+    """
+    When method POST
+    Then status 201
+
+    # 2. Create an order line
+    Given path 'orders/order-lines'
+    * def poLine = read('classpath:samples/mod-orders/orderLines/minimal-order-line.json')
+    * set poLine.id = poLineId3
+    * set poLine.purchaseOrderId = orderId3
+    * set poLine.fundDistribution[0].fundId = fundId
+    And request poLine
+    When method POST
+    Then status 201
+
+    # 3. Open the order
+    Given path 'orders/composite-orders', orderId3
+    When method GET
+    Then status 200
+
+    * def order = $
+    * set order.workflowStatus = 'Open'
+
+    Given path 'orders/composite-orders', orderId3
+    And request order
+    When method PUT
+    Then status 204
+
+    # 4. Receive the piece
+    # 4.1 Get the id of piece created when the order was opened
+    Given path 'orders/pieces'
+    And param query = 'poLineId==' + poLineId3
+    When method GET
+    Then status 200
+    And match $.totalRecords == 1
+    * def pieceId = $.pieces[0].id
+
+    # 4.2 Receive it
+    Given path 'orders/check-in'
+    And request
+    """
+    {
+      toBeCheckedIn: [
+        {
+          checkedIn: 1,
+          checkInPieces: [
+            {
+              id: "#(pieceId3)",
+              itemStatus: "In process",
+              locationId: "#(globalLocationsId)"
+            }
+          ],
+          poLineId: "#(poLineId3)"
+        }
+      ],
+      totalRecords: 1
+    }
+    """
+    When method POST
+    Then status 200
+    * call pause 500
+
+    # 5. Retrieve title and assign acqUnit2
+    * configure headers = headersAdmin
+    Given path 'orders/titles'
+    And param query = 'poLineId==' + poLineId3
+    When method GET
+    Then status 200
+    And match $.totalRecords == 1
+    * def titleResponse = $.titles[0]
+    * set titleResponse.acqUnitIds = ['#(acqUnitId2)']
+    * def titleId = $.titles[0].id
+
+    Given path 'orders/titles', titleId
+    And request titleResponse
+    When method PUT
+    Then status 204
+
+    # 6. Remove acqUnit 'acqUnitId2' membership, user only has 'acqUnitId1'
+    * configure headers = headersAdmin
+    Given path 'acquisitions-units/memberships', acqUnitMembershipId3
+    When method DELETE
+    Then status 204
+
+    # 7. Check piece 1 receivingStatus and update the piece should be forbidden
+    Given path 'orders/pieces', pieceId
```

Comment:
Is this should be pieceId3 instead of pieceId?

Reply:
Actually it should be pieceId previous operations changed to pieceId3 to pieceId, because of line 515

Reply:
plaese add comment about it